airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -120,9 +120,9 @@
             "properties": {
               "replication_type": {
                 "type": "string",
-                "const": "Standard",
-                "enum": ["Standard"],
-                "default": "Standard",
+                "const": "STANDARD",
+                "enum": ["STANDARD"],
```

Comment:
```suggestion
```
I a `const` is declared I think an `enum` is redundant.

Reply:
@alafanechere agree with you but our spec-linter expect to have the const/enum/default values.